asdf/schema.py

```diff
@@ -537,6 +527,7 @@ def get_validator(
     -------
     validator : jsonschema.Validator
     """
+    # TODO deprecate url_mapping
```

Comment:
I think we can just remove this as this function isn't part of the documented API. I would first like to double check it's not in use by our downstream and to handle this in a separate PR. I'll open and issue and remove this TODO as this PR progresses.

Reply:
https://github.com/asdf-format/asdf/issues/1864